db/system_keyspace_clients.cc

```diff
@@ -1,98 +0,0 @@
-/*
- * This file is part of Scylla.
- *
```

Comment:
I don't understand why I'm reviewing a file being added and deleted. 

Patchsets should be constructed in a way that is friendly to reviewers, not make them do extra work be reviewing everything several times and trying to reverse engineer what happened.

Reply:
My apologies - I thought that reviewers inspect the "Changed files" tab, while individual commits represent the "historical thought process" behind PR. Spurious changesets are now squashed away.